boilerplate/app/components/Screen.tsx

```diff
@@ -265,19 +265,28 @@ export function Screen(props: ScreenProps) {
         style={statusBarStyle || (themeContext === "dark" ? "light" : "dark")}
         {...StatusBarProps}
       />
-
-      <KeyboardAvoidingView
-        behavior={isIos ? "padding" : "height"}
-        keyboardVerticalOffset={keyboardOffset}
-        {...KeyboardAvoidingViewProps}
-        style={[$styles.flex1, KeyboardAvoidingViewProps?.style]}
-      >
-        {isNonScrolling(props.preset) ? (
+      {/**
+       * KeyboardAvoidingView crashes in web,
+       * therefore we want to use ScrollView just for web
+       */}
+      {Platform.OS === "web" ? (
+        <ScrollView>
           <ScreenWithoutScrolling {...props} />
-        ) : (
-          <ScreenWithScrolling {...props} />
-        )}
-      </KeyboardAvoidingView>
+        </ScrollView>
+      ) : (
+        <KeyboardAvoidingView
```

Comment:
Why do we even have KeyboardAvoidingView if we also have KeyboardAwareScrollView nested within it? If we need it for the non-scrolling version, I would think:

1. We wouldn't want to wrap KeyboardAvoidingView around both cases, only non-scrolling
2. We wouldn't want to mix RN's KeyboardAvoidingView with RNKC's KeyboardAwareScrollView. It looks like RNKC has its own KeyboardAvoidingView.